### PR TITLE
Add Spanish translations to exceptions

### PR DESCRIPTION
**Pull request by larzuk91**
Let's make this bundle a bit more complete by adding Spanish translations to the exception messages

### CODE REVIEW

File: src/Resources/translations/VerifyEmailBundle.es.xlf [hunk 1]
```diff
@@ -22,6 +22,22 @@
                 <source>%count% minute|%count% minutes</source>
                 <target>%count% minuto|%count% minutos</target>
             </trans-unit>
+            <trans-unit id="5">
+                <source>%count% minute|%count% minutes</source>
+                <target>%count% minute|%count% minutes</target>
+            </trans-unit>
```
Comment (jrushlow) [inline src/Resources/translations/VerifyEmailBundle.es.xlf:28]: I believe this is a duplicate of above?

```suggestion
```

**Review by jrushlow — CHANGES_REQUESTED** [review]
Thanks @larzuk91! I figured you would of had this in before Victor could get his in 😆

File: src/Resources/translations/VerifyEmailBundle.es.xlf (hunk 1 above)
Comment (larzuk91) [reply src/Resources/translations/VerifyEmailBundle.es.xlf]: Woops! Sorry, my copy-paste skills are the worst :D

**Comment by larzuk91** [comment]
> Thanks @larzuk91! I figured you would of had this in before Victor could get his in 😆

hahaha, he bested me at this :)

**Review by jrushlow — APPROVED** [review]

**Pull request merged**